src/connection-monitor-preview/azext_connection_monitor_preview/azext_metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.0.80",
```

Comment:
If we set this to 2.0.80, better to let user upgrade their CLI in our readme.md

Reply:
Got it.